src/content/en/2019/third-parties.md

```diff
@@ -101,30 +104,32 @@ As for specific third-party providers, the same juggernauts topping the request
+
 <insert table of metric 05_10>
 
-While much could be said about every individual provider’s popularity and performance impact, this more opinionated analysis is left as an exercise for the reader and other purpose-built tools such as [third-party-web](https://thirdpartyweb.today).
+While much could be said about every individual provider’s popularity and performance impact, this more opinionated analysis is left as an exercise for the reader and other purpose-built tools such as the previously mentioned [third-party-web](https://thirdpartyweb.today).
 
 ## Analysis
 
 ### Usage Patterns
 
-Why do site owners use third-party code? How did third-party content grow to be nearly half of all network requests? What is all of this code doing? Answers to these questions lie in the three primary usage patterns of third-party resources. Broadly, site owners reach for third parties to generate and consume data from their users, monetize their site experiences, and simplify web development.
+Why do site owners use third-party code? How did third-party content grow to be nearly half of all network requests? What are all these requests doing? Answers to these questions lie in the three primary usage patterns of third-party resources. Broadly, site owners reach for third parties to generate and consume data from their users, monetize their site experiences, and simplify web development.
 
 ### Generate and Consume Data
 
 Analytics is the most popular third-party category found across the web and yet is minimally user-visible. Consider the volume of information at play in the lifetime of a web visit; there’s user context, device, browser, connection quality, location, page interactions, session length, return visitor status, and more being generated continuously. It’s difficult, cumbersome, and expensive to maintain tools that warehouse, normalize, and analyze time series data of this magnitude. While nothing categorically necessitates that analytics fall into the domain of third-party providers, the widespread attractiveness of understanding your users, deep complexity of the problem space, and increasing emphasis on managing data respectfully and responsibly naturally surfaces analytics as a popular third-party usage pattern.
 
 There’s also a flip side to user data though: consumption. While analytics is about generating data from your site’s visitors, other third-party resources focus on consuming data about your visitors that is known only by others. Social providers fall squarely into this usage pattern. A site owner _must_ use Facebook resources if they wish to integrate information from a visitor’s Facebook profile into their site. As long as site owners are interested in personalizing their experience with widgets from social networks and leveraging the social networks of their visitors to increase their reach, social integrations are likely to remain the domain of third-party entities for the foreseeable future.
 
-### Monetize
+### Monetization
```

Comment:
The title of this was `Monetize` to be parallel with the other usage patterns which are also actions. They all complete the sentence 

"Developers use third parties in order to...."

Would it be better to align *all* of them on the noun forms? `Generation and Consumption of Data`, `Simplification of Development`

Reply:
I'm not sure to be honest and struggled with this one to be honest but Monetize sounded a little wrong - like most native English speakers my grammar sucks I go with the "it sounds right or wrong" rather than any explicit grammatical rules rules ;-) I'm easy if we leave as is or if you think better to change the others?

Reply:
What about "Monetize Web Traffic"? Or "Monetize Visits" Re-reading it, I think what I didn't like was that it was a big fragmented when it was just "Monetize" on it's own.

Reply:
Monetize Web Traffic 👍 perfect!